setup.py

```diff
@@ -224,7 +224,7 @@ def make_distribution(self):
     ],
     install_requires=[
             "six",
-            "toml",
+            "tomli",
```

Comment:
Could we use the built-in [`tomllib`](https://docs.python.org/3/library/tomllib.html) on Python 3.11 and newer instead?

```suggestion
            "tomli; python_version<'3.11'",
```